frontend/Views/Widgets/ViewNemesisStats.svelte

```diff
@@ -0,0 +1,312 @@
+            updateCharts(filteredData);
+        }
+        loading = false;
+    });
+</script>
+
+<div class="container-fluid p-3">
+    {#if errorMsg}
+        <div class="alert alert-danger" role="alert">
+            {errorMsg}
+        </div>
+    {/if}
+    {#if loading}
+        <div class="text-center my-3">
+            <span>Loading...</span>
+        </div>
+    {/if}
+    <div class="row mb-4">
+        <div class="col">
+            <div class="card">
+                <div class="card-header">
+                    <h5 class="card-title mb-0">Nemesis Statistics</h5>
+                </div>
+                <div class="card-body">
+                    <div class="btn-group mb-3" role="group">
+                        <button
+                            class="btn btn-outline-secondary {selectedFilter === '' ? 'active' : ''}"
+                            on:click={() => applyFilter("")}>All</button
+                        >
+                        {#each releases as release}
+                            <button
+                                class="btn btn-outline-secondary {selectedFilter === release ? 'active' : ''}"
+                                on:click={() => applyFilter(release)}>{release}</button
+                            >
+                        {/each}
+                    </div>
+                    <div class="btn-group d-block mb-3" role="group">
+                        {#each versions as version}
+                            <button
+                                class="btn btn-outline-primary btn-sm {selectedFilter === version ? 'active' : ''}"
+                                on:click={() => applyFilter(version)}>{version}</button
+                            >
+                        {/each}
+                    </div>
+
+                    <div class="row">
+                        <div class="col-md-3">
+                            <div class="small-chart-container mb-3">
+                                <canvas bind:this={chartCanvas} />
+                            </div>
+                            <div class="alert alert-info text-center py-2">
+                                <small>Total Duration: {(totalDuration / 3600).toFixed(2)} hours</small>
+                            </div>
+                            <div class="alert alert-secondary text-center py-2 my-2">
+                                <small>Unique Nemesis Types: {uniqueNemesisCount}</small>
+                            </div>
+                        </div>
+                        <div class="col-md-9">
+                            <div class="bar-chart-container">
+                                <canvas bind:this={barChartCanvas} />
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<style>
+    .small-chart-container {
+        width: 100%;
+        height: 300px;
```

Comment:
Have you checked other resolutions and screen sizes? Might need some breakpoints here